### PR TITLE
OCPQE-16287: Alter cron for baremetal ec jobs 

### PR DESCRIPTION
**Pull request by dis016**
[OCPQE-16287](https://issues.redhat.com//browse/OCPQE-16287):  Alter cron for baremetal ec jobs to keep resources available for nightly jobs.

/cc @liangxia @jhou1 @pruan-rht

### CODE REVIEW

**Comment by pruan-rht** [comment]
@dis016 how do you determine the new launch frequency for the jobs?  Did you do it manually or use [this](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh) script

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-ec.yaml [hunk 1]
```diff
@@ -1579,7 +1579,7 @@ tests:
     workflow: baremetal-lab-agent-install-pxe
 - as: baremetal-pxe-ha-agent-ipv4-static-connected-f14
   cluster: build05
-  cron: 30 14 5,19 * *
+  cron: 30 14 11,26 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-ec.yaml:1582]: This one and the above one are running at the same date/time.

**Comment by dis016** [comment]
> 

@pruan-rht i had manually altered the cron, as those ec jobs are disturbing the nightly

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-ec.yaml (hunk 1 above)
Comment (dis016) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-ec.yaml:1582]: @liangxia i had observed other baremetal jobs, and looks 11,26 is having less jobs. so used it twice for ec jobs.

**Comment by dis016** [comment]
@pruan-rht @liangxia @jhou1 updated the values as per generate-cron-entry.sh script

**Comment by jhou1** [comment]
/lgtm

Thanks @dis016

**Pull request merged**